sci-libs/nipype/nipype-9999.ebuild

```diff
@@ -25,6 +25,7 @@ DEPEND="
 	dev-python/numpy[${PYTHON_USEDEP}]
 	sci-libs/nibabel[${PYTHON_USEDEP}]"
 RDEPEND="
+	dev-lang/python[sqlite]
```

Comment:
@jlec this is already pulled - I hope [this fix](https://github.com/gentoo-science/sci/pull/459) is ok.